### PR TITLE
add instructions for linking

### PR DESCRIPTION
**Pull request by futurepaul**

### CODE REVIEW

**Review by benthecarman — APPROVED** [review]

File: justfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set dotenv-load := false
+
```
Comment (benthecarman) [inline justfile:2]: would be nice to have a run command too

**Comment by benthecarman** [comment]
Doesn't seem to work for me

**Comment by futurepaul** [comment]
did you use the new instruction here? https://github.com/MutinyWallet/mutiny-node/pull/394

and are your `mutiny-node` and `mutiny-web` folders siblings?

**Comment by benthecarman** [comment]
> and are your mutiny-node and mutiny-web folders siblings?

they are both in `~/projects`

however `mutiny-wasm` is inside of mutiny-node, so idk if that would break something?

**Comment by futurepaul** [comment]
no, that's the same for me

**Review by benthecarman — APPROVED** [review]

**Pull request merged**